### DISCUSSION TITLE
Holy Unblocker

### DISCUSSION THREAD

**Discussion by Endernates** [3kh0/ext-remover#295] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I tried to use the holy unblocker and it Brought me to a page that says Application error

**Comment by fnfgo** [comment, marked as answer]
yea thats normal bitch

**Reply by Endernates** [reply]
Thanks 😄✨

**Reply by Theblackmamba824** [reply]
no it isnt

lemme tell u why. so basically herokuapp.com is not free any more and so the people who dont have a membership or whatever there websites got closed

**Reply by fnfgo** [reply]
bro thats what i meant jeez.😑

**Reply by Theblackmamba824** [reply]
🤣🤣🤣🤣🤣

**Reply by Aka-but-nice** [reply]
Here's a link for Holy Unblocker, "https://aadynpersonalunblocker.glitch.me/" also if you want Roblox use this proxy and go to https://kxd.fm/roblox/ it should work YOU HAVE TO USE A proxy for https://kxd.fm/roblox/ if you try it in your regular browser it will load up, but the game will not so you have to use that proxy!

**Reply by fnfgo** [reply]
> https://kxd.fm/roblox/

these sites are usally blocked render and heroku are the only unblocked for me

**Reply by Aka-but-nice** [reply]
use that link here "https://theoxymoron.xyz/#"

**Reply by Theblackmamba824** [reply]
> > https://kxd.fm/roblox/
> 
> these sites are usally blocked render and heroku are the only unblocked for me

wuts render

**Reply by Endernates** [reply]
https://pineappleeater.glitch.me/1.html

**Comment by Aka-but-nice** [comment]


**Reply by fnfgo** [reply]
chrome://flags is usually blocked idiot

**Reply by bartholemuew** [reply]
damn bro chill

**Reply by Aka-but-nice** [reply]
Bro, I'm not an idiot, but if chrome://flags is blocked, ask the teacher to update your Chromebook, or update it manually! ALSO, check out my site "best-site-everr.glitch.me"!

**Reply by bartholemuew** [reply]
> Bro, I'm not an idiot, but if chrome://flags is blocked, ask the teacher to update your Chromebook, or update it manually! ALSO, check out my site "best-site-everr.glitch.me"!

nice site

**Reply by fnfgo** [reply]
Glitch is also blocked

**Reply by fnfgo** [reply]
![image](https://user-images.githubusercontent.com/113545974/206731263-eae5c214-69e1-4101-a531-0d8fc707e18a.png)

**Reply by fnfgo** [reply]
> chrome://flags is usually blocked idiot

![image](https://user-images.githubusercontent.com/113545974/206731798-3000bf65-7797-4a44-b59a-b74f55d7ea9a.png)

**Reply by fnfgo** [reply]
> I tried to use the holy unblocker and it Brought me to a page that says Application error

sry man i got Little mad

**Comment by rwg100** [comment]
> chrome://flags is usually blocked idiot
> yea thats normal bitch

i cannot be the one who hates this kid bro

**Reply by Aka-but-nice** [reply]
You ain't tbh
